code_review.md

```diff
@@ -4,109 +4,53 @@ Code review
-
-Unclear commit summaries:
-> Fixed some normalization bugs
-
-> Bugfixes
-
-The unclear messages make it hard to tell at a glance what changed, and that makes browsing the commit history harder.
-
-### Commit messages should be as detailed as they need to be (and no more)
-
-The commit message is the rest of the commit past the first line.
-If a commit makes a small and obvious change, it's fine to not even have a commit message past the summary.
-
-The commit message is your place to clarify the justification for a change.
-While there's no need to rehash anything that code comments already say, if there's more detail that helps a reader understand *why* a change was made, be as verbose as you need to!
-Remember: future-you (or another developer) will read this when going through the commit history to understand why a change was made.
-Make their life easier.
+The contribution process in general is outlined in the [contributing document](CONTRIBUTING.md).
+This provides a checklist and set of guidelines for what to do when doing code review.
+It also helps set expectations for contributors about what to expect.
+
+Checklist
+---------
+
+### Meta
+- [ ] Have they submitted a signed contributor's agreement?
+- [ ] Does this meet the requirements? Does it fix the bug?
+- [ ] Is this feature useful to multiple users or potential users?
+- [ ] Is this one logical piece of work? Does it make sense to be split into multiple PRs for each sub-feature?
+- [ ] Is there documentation?  This includes docstrings, comments & potentially user-facing documentation.
+- [ ] Is the commit history clean?
+
+### Architecture
+- [ ] Does this duplicate functionality found elsewhere?
+- [ ] Does it make more sense for this to be implemented elsewhere? E.g. Should this happen in the storage service vs a client script, or should a helper function live in archivematicaCommon?
+- [ ] Does this add or reduce technical debt?
+- [ ] Does this conflict with or benefit from work being done in parallel?
+- [ ] Does this have security implications? Does it add or close a security vulnerability?
+- [ ] Is there a more efficient way to do this? Is the more efficient way significantly less readable?
+- [ ] Is this change backwards compatible? If not, could it be?  Should the API be versioned?
+- [ ] Does this work for upgrades?  Does this require additional processing for existing items?
+
+### Style
+- [ ] Does it follow [PEP8](https://www.python.org/dev/peps/pep-0008/) (code style) and [PEP257](https://www.python.org/dev/peps/pep-0257/) (docstrings)?
+- [ ] Do docstrings list the parameters and behaviour?
+- [ ] Are the names accurate and sensible?
+- [ ] Are there massive functions, classes or files? Can they be split?
+- [ ] Are errors handled?  Is the error handling consistent with similar code?
+
+### Syntax:
+- [ ] Is it Python 2.7 & Python 3 compatible?
+- [ ] Are all user-facing strings marked for translation?
```

Comment:
This may lead to confusion because we haven't merged the i18n work yet but it should be happening very soon so I would just leave it! Thanks.